### PR TITLE
feat(@vtmn/svelte): add badge component

### PR DESCRIPTION
**Pull request by Tlahey**
## Changes description

- Add `badge` component in svelte

## Checklist

- [X] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [X] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [X] Check your code additions will fail neither code linting checks.
- [X] I have reviewed the submitted code.
- [X] I have tested on related showcases.
- [X] If it includes design changes, please ask for a review with a core team designer.

## Does this introduce a breaking change?

- No

### CODE REVIEW

File: packages/sources/svelte/src/components/indicators/VtmnBadge/VtmnBadge.svelte
Comment (GaspardMathon) [inline packages/sources/svelte/src/components/indicators/VtmnBadge/VtmnBadge.svelte:20]:
```suggestion
  /**
   * The value of the badge
   * @type {'default' | 'brand' | 'reversed' | 'accent' | 'alert'}
   */
  export let variant = VARIANT.DEFAULT;
  
   let className;
  /**
   * @type {string} Custom classes to apply to the component.
   */
  export { className as class };
```

File: packages/sources/svelte/src/components/indicators/VtmnBadge/VtmnBadge.svelte
Comment (GaspardMathon) [inline packages/sources/svelte/src/components/indicators/VtmnBadge/VtmnBadge.svelte:31]: I think it would be clearer if all the controls are processed in line 22.

**Review by GaspardMathon — CHANGES_REQUESTED** [review]

File: packages/showcases/svelte/stories/components/indicators/VtmnBadge/docs.mdx
Comment (lauthieb) [inline packages/showcases/svelte/stories/components/indicators/VtmnBadge/docs.mdx:51]: Interesting :)

**Review by lauthieb — APPROVED** [review]
Thanks a lot @Tlahey for this contribution!
I'm waiting for the new review of @GaspardMathon, if it's ok after its old remarks, we will merge!

File: packages/showcases/svelte/stories/components/indicators/VtmnBadge/docs.mdx
Comment (Tlahey) [reply packages/showcases/svelte/stories/components/indicators/VtmnBadge/docs.mdx:51]: We need to rollback storybook svelte to v6.3.13, Storybook have an issue with the generated template of svelte on ^6.4.
After the rollback, we can change the documentation to be more clear and commit the PR :)

**Review by lauthieb — CHANGES_REQUESTED** [review]

**Review by lauthieb — APPROVED** [review]

**Pull request merged**
